registry/recipes/mirror.md

```diff
@@ -63,34 +63,40 @@ Multiple registry caches can be deployed over the same back-end.  A single
 registry cache will ensure that concurrent requests do not pull duplicate data,
 but this property will not hold true for a registry cache cluster.
 
-### Configuring the cache
+### Configure the cache
 
 To configure a Registry to run as a pull through cache, the addition of a
 `proxy` section is required to the config file.
 
 In order to access private images on the Docker Hub, a username and password can
 be supplied.
 
-    proxy:
-      remoteurl: https://registry-1.docker.io
-      username: [username]
-      password: [password]
+```yaml
+proxy:
+  remoteurl: https://registry-1.docker.io
+  username: [username]
+  password: [password]
+```
 
-> :warn: if you specify a username and password, it's very important to understand that private resources that this user has access to on the Hub will be made available on your mirror. It's thus paramount that you secure your mirror by implementing authentication if you expect these resources to stay private!
+> **Warning**: If you specify a username and password, it's very important to
+> understand that private resources that this user has access to on the Hub will
```

Comment:
on the Hub -> on Docker Hub (maybe Docker Store, but I'm not sure we really made that transition yet)